Adyen/UI/View Controllers/SecuredViewController.swift

```diff
@@ -78,7 +78,7 @@ public final class SecuredViewController: UIViewController {
         delegate?.viewWillAppear(viewController: self)
     }
 
-    public func getChildViewController() -> UIViewController {
+    public var getChildViewController: UIViewController {
```

Comment:
why not just expose the childViewController and make it internal or something? :-) instead of this var, since getXX is not a good Swift var name